types/cytoscape-fcose/index.d.ts

```diff
@@ -0,0 +1,126 @@
+// Type definitions for cytoscape-fcose 2.1
+// Project: https://github.com/iVis-at-Bilkent/cytoscape.js-fcose
+// Definitions by: Paul Paulsen <https://github.com/Phpaulsen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import cytoscape = require('cytoscape');
+
+declare const cytoscapeFcose: cytoscape.Ext;
+
+export = cytoscapeFcose;
+export as namespace cytoscapeFcose;
+
+declare namespace cytoscapeFcose {
+    interface FcoseFixedNodeConstraint {
+        nodeId: string;
+        position: cytoscape.Position;
+    }
+
+    interface FcoseAlignmentConstraint {
+        vertical: Array<[string, string]>;
+        horizontal: Array<[string, string]>;
+    }
+
+    interface FcoseRelativeVerticalPlacementConstraint {
+        top: string;
+        bottom: string;
+        gap: number;
+    }
+
+    interface FcoseRelativeHorizontalPlacementConstraint {
+        left: string;
+        right: string;
+        gap: number;
+    }
+
+    type FcoseRelativePlacementConstraint =
+        | FcoseRelativeVerticalPlacementConstraint
+        | FcoseRelativeHorizontalPlacementConstraint;
+
+    interface FcoseLayoutOptions extends cytoscape.BaseLayoutOptions {
+        name: 'fcose';
+
+        // 'draft', 'default' or 'proof'
+        // - "draft" only applies spectral layout
+        // - "default" improves the quality with incremental layout (fast cooling rate)
+        // - "proof" improves the quality with incremental layout (slow cooling rate)
+        quality?: "default" | "draft" | "proof";
+        // Use random node positions at beginning of layout
+        // if this is set to false, then quality option must be "proof"
+        randomize?: boolean;
+        // Whether or not to animate the layout
+        animate?: boolean;
+        // Duration of animation in ms, if enabled
+        animationDuration?: number;
+        // Easing of animation, if enabled
+        animationEasing?: undefined | "ease-out";
```

Comment:
```suggestion
        animationEasing?: "ease-out" | undefined;
```
just a convention here